test/libsolidity/semanticTests/array/concat/bytes_concat_empty.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+    function f() public returns (bytes memory) {
+        return bytes.concat();
```

Comment:
I added slightly different one, because this one is perfectly fine. `bytes.concat` is of a function type. 
So I tried returning it in function that returns `bytes memory`, so it fails type checker.